conda-forge.yml

```diff
@@ -7,3 +7,5 @@ github:
 provider:
   linux_aarch64: default
   linux_ppc64le: default
+azure:
+  store_build_artifacts: True
```

Comment:
Would make sure to drop this and re-render before merging (once we are ready for that)

```suggestion
```